bpf/http_defs.h

```diff
@@ -0,0 +1,19 @@
+#ifndef HTTP_DEFS_H
+#define HTTP_DEFS_H
+
+#define MAX_CONCURRENT_REQUESTS 10000
```

Comment:
Various header constants that appear in Linux headers, related to parsing sock and socket structs.